Documentation/ceph-disaster-recovery.md

```diff
@@ -231,9 +237,97 @@ kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=1
 
 The operator will automatically add more mons to increase the quorum size again, depending on the `mon.count`.
 
-# Adopt an existing Rook Ceph cluster into a new Kubernetes cluster
+## Restoring CRDs After Deletion
+
+When the Rook CRDs are deleted, the Rook operator will respond to the deletion event to attempt to clean up the cluster resources.
+If any data appears present in the cluster, Rook will refuse to allow the resources to be deleted since the operator will
+refuse to remove the finalizer on the CRs until the underlying data is deleted. For more details, see the
+[dependency design doc](https://github.com/rook/rook/blob/master/design/ceph/resource-dependencies.md).
+
+While it is good that the CRs will not be deleted and the underlying Ceph data and daemons continue to be
+available, the CRs will be stuck indefinitely in a `terminating` state in which the operator will not
```

Comment:
They don't go into this state until the resource is actually being deleted by kubernetes. I think it should be safe to call this `Deleting` since that is Rook's applied state when deleting a resource.

```suggestion
available, the CRs will be stuck indefinitely in a `terminating` state in which the operator will not
```